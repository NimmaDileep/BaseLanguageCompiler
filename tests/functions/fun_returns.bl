struct Point {
    x: Int,
    y: Int
}

fun x() -> Int {
    return 1;
}

fun y() -> String {
    return "dog";
}

fun z() -> Boolean {
    return true;
}

fun pt() -> Point {
    return Point(1, 2);
}

fun arr() -> [Int] {
    return [1, 2, 3];
}

print(x());
print(y());
print(z());
print(pt());
print(arr());
