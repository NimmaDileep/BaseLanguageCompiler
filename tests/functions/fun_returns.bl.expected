1
dog
true
Point{x=1,y=2}
[1, 2, 3]
